docs/Developer-Guide_Build-Preparation.md

```diff
@@ -42,8 +42,10 @@ Make sure that full path to the build script **does not contain spaces**.
 
 ## Providing build configuration
 
-After the first run of `compile.sh` a new configuration file `config-example.conf` and symlink `config-default.conf` will be created.
-You may edit it to your needs or create different configuration files using it as a template.
+After the first run of `compile.sh` a new configuration file `config-example.conf` and symlink `config-default.conf` will be created in `userpatches`.
```

Comment:
This is not real. No examples, symlinks, or anything like should be created by the build system, and if they are, it's a bug.

Reply:
This was MO with old system, so this info is deprecated.

Reply:
I just ever saw it cos the PR touched it ;-)